schedule/schedule-appengine/src/test/java/com/jasify/schedule/appengine/model/SchemaMigrationTest.java

```diff
@@ -29,6 +29,7 @@
     public void initializeDatastore() {
         TestHelper.initializeDatastore();
         ApplicationData.instance().reload();
+        UniqueConstraints.ensureAllConstraintsExist();
```

Comment:
Dont think you need this....   
 public static void initializeDatastore() {
         initializeDatastore(datastoreHelper);
     }
 
     public static void initializeDatastore(LocalServiceTestHelper datastoreHelper) {
         datastoreHelper.setUp();
         UserContext.clearContext();
+        UniqueConstraints.ensureAllConstraintsExist();
     }